src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -235,7 +234,14 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     }
     else
     {
-      macroSubstitute(&payload, httpInfo.payload, ce);
+      bool b = macroSubstitute(&payload, httpInfo.payload, ce);
+
+      if (b == false)
+      {
+        LM_E(("Bad Input (not sending NotifyContextRequest: payload too large)"));
```

Comment:
I think is better to use LM_W. It is not a bug in CB, but a kind of consequence of bad usage on the users side.